docs/mirror.md

```diff
@@ -0,0 +1,202 @@
+# Create a local mirror of a Percona repository
+
+Enterprises introduce various security policies: for example, allow Internet access only on one server or deny root privileges to database administrators. 
+
+To comply with these policies and install Percona software in your environment, a solution is to set up the local mirror of Percona repositories. This document provides instructions how to do it.
+
+!!! important
+
+    The instructions below are for AMD/Intel x64 architectures. 
+
+## Prerequisites
+
+To create a local mirror of a Percona repository, you need the following software:
+
+* Rsync to download packages from Percona repositories and upload them to your mirror
+* Repository creation tools:
+
+    - `dpkg-dev` for Debian and Ubuntu
+    - `createrepo` for RHEL and derivatives
+
+## Debian and Ubuntu
+
+1. Install `dpkg-dev` utility to unpack, build and upload Debian source packages.
+
+    ```{.bash data-prompt="$"}
+    $ sudo apt install dpkg-dev
+    ```
+
+2. Create the folder where your mirror repository resides. For example, `/opt/debs/`
+
+    ```{.bash data-prompt="$"}
+    $ sudo mkdir -p /opt/debs/
```

Comment:
you need to add additional permissions to this directory as you create it under root and if different user would try to copy/create file inside it - the result will be "permission deny"